dashboards/blazor-server/blazor-server.json

```diff
@@ -0,0 +1,935 @@
+						"text": "![Blazor logo](https://www.pragimtech.com/wp-content/uploads/2020/03/blazor-tutorial-for-beginners.png)"
+					}
+				},
+				{
+					"title": "Application (click on the application name to filter)",
+					"layout": {
+						"column": 3,
+						"row": 2,
+						"width": 3,
+						"height": 4
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.bar"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountId": 0,
+								"query": "FROM Metric SELECT count(*) FACET appName "
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "Average Load Time (seconds)",
+					"layout": {
+						"column": 6,
+						"row": 2,
+						"width": 5,
+						"height": 4
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.line"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"legend": {
+							"enabled": true
+						},
+						"nrqlQueries": [
+							{
+								"accountId": 0,
+								"query": " FROM Transaction SELECT sum(duration)/count(*) AS 'Average Duration (seconds)', count(*) FACET request.headers.referer TIMESERIES AUTO"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						},
+						"yAxisLeft": {
+							"zero": true
+						}
+					}
+				},
+				{
+					"title": "Live Connections",
+					"layout": {
+						"column": 11,
+						"row": 2,
+						"width": 2,
+						"height": 2
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountId": 0,
+								"query": "SELECT count(*) AS '' FROM Transaction WHERE request.headers.connection LIKE '%keep-alive%'"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						},
+						"thresholds": [
+							{
+								"value": 0
+							}
+						]
+					}
+				},
+				{
+					"title": "",
+					"layout": {
+						"column": 1,
+						"row": 4,
+						"width": 2,
+						"height": 2
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.markdown"
+					},
+					"rawConfiguration": {
+						"text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/catalog-pack-details?state=ea3bfa18-653b-4679-b69f-5c02b49e903d)\n\nUnable to find data in your dashboard? - [Troubleshoot here](\nhttps://docs.newrelic.com/docs/apm/agents/net-agent/installation/introduction-net-agent-install/)\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=BlazorServer) here and let us know how we can improve it for you.\n"
```

Comment:
please update the link on Add Data - https://docs.newrelic.com/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework/#technologies

Reply:
Hi @pkosanam ,
Updated doc's URL in the dashboard. Could you review it?